src/torque-slides.jsx

```diff
@@ -32,6 +40,23 @@ var TorqueSlides = React.createClass({
     this.setInterval(this.updateSlide, 1000);
   },
 
+  keydown(e) {
+    switch (e.keyCode) {
+      case RIGHT_ARROW:
+        this.setState({
+          current: (this.state.current + 1) % this.props.children.length,
+          countdown: this.props.duration
```

Comment:
I think this needs to take into account the possibility of the next slide having a custom duration.  Look at the updateSlide function.